debian/lxc-android-config.upstart

```diff
@@ -29,9 +29,16 @@ post-start script
             sleep 0.1
         done
 
+        # If the socket isn't available, 'getprop' falls back to reading files
+        # manually, causing a false positive of propertyservice being up
+        while [ ! -e /dev/socket/property_service ]; do sleep 0.1; done
+
+        # This property is read by libhybris itself to pick a linker, so it
+        # *MUST* be available before we emit 'android'.
+        while [ -z $(getprop ro.build.version.sdk) ]; do sleep 0.1; done
```

Comment:
Please quote the shell substitute to be more resilient. ([SC2046](https://github.com/koalaman/shellcheck/wiki/SC2046))

```suggestion
        while [ -z "$(getprop ro.build.version.sdk)" ]; do sleep 0.1; done
```